docs/en/manage/tuning-for-cloud-cost-efficiency.md

```diff
@@ -19,7 +19,7 @@ However, you can use a higher rate of insert queries per second when you use asy
 
 ## Use asynchronous inserts 
 
-Use [asynchronous inserts](https://clickhouse.com/blog/click-house-v2111-released) as an alternative to both batching data on the client-side and keeping the insert rate at around one insert query per second by enabling the [async_insert](../operations/settings/settings/#async-insert) setting. This causes ClickHouse to handle the batching on the server-side. Doing so will therefore reduce the number of write requests generated.
+Use [asynchronous inserts](https://clickhouse.com/blog/click-house-v2111-released) as an alternative to both batching data on the client-side and keeping the insert rate at around one insert query per second by enabling the [async_insert](../../operations/settings/settings#async-insert) setting. This causes ClickHouse to handle the batching on the server-side. Doing so will therefore reduce the number of write requests generated.
```

Comment:
```suggestion
Use [asynchronous inserts](https://clickhouse.com/blog/click-house-v2111-released) as an alternative to both batching data on the client-side and keeping the insert rate at around one insert query per second by enabling the [async_insert](/docs/en/operations/settings/settings.md/#async-insert) setting. This causes ClickHouse to handle the batching on the server-side. Doing so will therefore reduce the number of write requests generated.
```

Reply:
having the file extension `.md` causes Docusaurus to verify the link, and having the full path to the file (starting with `/docs`) gives us a better chance of avoiding 404's on page refresh.